metpy/calc/thermo.py

```diff
@@ -513,3 +513,114 @@ def equivalent_potential_temperature(pressure, temperature):
     pottemp = potential_temperature(pressure, temperature)
     smixr = saturation_mixing_ratio(pressure, temperature)
     return pottemp * np.exp(Lv * smixr / (Cp_d * temperature))
+
+
+@exporter.export
+def virtual_temperature(temperature, mixing, molecular_weight_ratio=epsilon):
+    r"""Calculate virtual temperature.
+
+    This calculation must be given an air parcel's temperature and mixing ratio.
+    The implementation uses the formula outlined in [8]_.
+
+    Parameters
+    ----------
+    temperature: `pint.Quantity`
+        The temperature
+    mixing : `pint.Quantity`
+        dimensionless mass mixing ratio
+    molecular_weight_ratio : `pint.Quantity` or float, optional
+        The ratio of the molecular weight of the constituent gas to that assumed
+        for air. Defaults to the ratio for water vapor to dry air
+        (:math:`\epsilon\approx0.622`).
+
+    Returns
+    -------
+    `pint.Quantity`
+        The corresponding virtual temperature of the parcel
+
+    Notes
+    -----
+    .. math:: T_v = T \frac{\textsl{w} + \epsilon}{\epsilon\,(1 + \textsl{w})}
```

Comment:
Would you mind changing the \textsl to \text? The documentation build doesn't know \textsl and it renders poorly on the online docs. Thanks!